substrate/frame/support/procedural/src/pallet/parse/mod.rs

```diff
@@ -558,6 +565,7 @@ mod keyword {
 	syn::custom_keyword!(pallet);
 	syn::custom_keyword!(extra_constants);
 	syn::custom_keyword!(composite_enum);
+	syn::custom_keyword!(view_functions);
```

Comment:
Should this be called `experimental_view_functions` (or `view_functions_experimental` to be consistent with `tasks_experimental` above)?

With this, we can aim to merge this and then address any nits in followups without worrying about breaking anything that might end up in a release.

Reply:
Good idea. Done